corona-warn-app/cwa-app-android
# PDF Generator (EXPOSUREAPP-9191)

~~- Implementation is not yet linked to UI, it will be possible after #4006 is merged~~
### Testing
1. generate and add some tc, vc, rc
2. open certificate detail and click "Druckversion anzeigen" from top app bar menu
~~3. check created pdf document in /data/data/de.rki.coronawarnapp.test/cache/test.pdf~~
3. click "Weiter"
4. check the PDF preview, try to share/print it and check formatting and data in the file

TODO:
- [x]  wire with UI after #4006 is merged
